distributions/stacks/openshift/application/kfp-tekton/kustomization.yaml

```diff
@@ -2,23 +2,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 namespace: kubeflow
```

Comment:
```suggestion
```
The Tekton deployment from kfp-tekton is designed to deploy in the `tekton-pipelines` namespace, so we don't want to override it in this kustomization.

Most of the Tekton documentation is assuming it deploys in `tekton-pipelines`. And many Tekton community extensions such as tektoncd/experimental and tektoncd/catalog have this assumption as well. Also the Tekton ko and releases are based on `tekton-pipelines` namespace which helps us easier to update Tekton.